### PR TITLE
Add missing assets for new categories

### PR DESCRIPTION
**Pull request by chrmod**

### CODE REVIEW

File: _locales/en/messages.json [hunk 1]
```diff
@@ -242,8 +242,13 @@
 		"message": "Enables websites to publish, distribute, and optimize audio and video content."
 	},
 	"category_cdn": {
+		"description": "Ghostery organizes tags by category. This is one of several tag categories.",
 		"message": "CDN"
 	},
+	"category_cdn_desc": {
+		"description": "User-facing description for the Social Media tag category.",
```
Comment (philipp-classen) [inline _locales/en/messages.json:249]: Most new categories have the "User-facing description for the Social Media tag category" description. Is this intended?

If a generic one is sufficient, perhaps "Ghostery organizes tags by category. This is one of several tag categories" (as in the others)?

Comment (chrmod) [reply _locales/en/messages.json:249]: this is a translation key for a description, not for the category name

**Review by smalluban — APPROVED** [review]
What about the translations to other languages?

File: _locales/en/messages.json (hunk 1 above)
Comment (philipp-classen) [reply _locales/en/messages.json:249]: But still, does it make sense to use "Social Media tag category" everywhere for the new ones? I don't see the link to the CDN or telemetry category?

For the others that we had before, it made more sense, for example:

```
"category_advertising_desc": {
		"description": "User-facing description for the Advertising tag category.",`
```

Comment (chrmod) [reply _locales/en/messages.json]: oh, now I see.

**Review by philipp-classen — APPROVED** [review]

**Pull request merged**